content/docs/0.9.x/reference/bridge/_index.md

```diff
@@ -129,12 +129,20 @@ The data is fetched from your Kubernetes cluster and shows useful information li
 link="./assets/uniform-view.png"
 caption="Keptn Bridge Uniform View">}}
 
-By selecting one integration, events on error level as well as sequence failures can be viewed. If there are new error events since the last time the integration was viewed, an indicator shows up next to them. When a sequence in this particular scope failed, then there is
+By selecting one integration, subscriptions and events on error level as well as sequence failures can be viewed. Subscriptions are used to make some integrations listen for specific events.
```

Comment:
You can view the related subscriptions, events on error level, and sequence failures by selecting an integration. It is also possible to directly configure the subscriptions to listen to specific events.